jwst/datamodels/schemas/multiexposure.schema.yaml

```diff
@@ -26,10 +26,28 @@ allOf:
               fits_hdu: ERR
               default: 0.0
               datatype: float32
+            wavelength:
+              fits_hdu: WAVELENGTH
+              datatype: float32
+            barshadow:
+              fits_hdu: BARSHADOW
+              datatype: float32
             area:
               fits_hdu: AREA
               default: 0.0
               datatype: float32
+            var_poisson:
+              fits_hdu: VAR_POISSON
+              datatype: float32
+            var_rnoise:
+              fits_hdu: VAR_RNOISE
+              datatype: float32
+            var_flat:
+              fits_hdu: VAR_FLAT
+              datatype: float32
+            pathloss:
+              fits_hdu: PATHLOSS
+              datatype: float32
```

Comment:
I would point to `variance.schema.yaml` here instead of listing them separately.  That said, in the variance schema, there are defaults defined.  Might want to double-check the behavior.